perf/lua/column_scan.lua

```diff
@@ -147,8 +168,9 @@ end
 do_or_die(box.once, 'init', function()
     log.info('Creating the test space...')
     local format = {}
-    for i = 1, params.column_count do
-        table.insert(format, {'field_' .. i, 'unsigned'})
+    table.insert(format, {'field_1', 'unsigned'})
+    for i = 2, params.column_count do
+        table.insert(format, {'field_' .. i, params.field_type})
```

Comment:
Is there any point in having 100 string columns when you always access only the first one? Maybe, better ignore `column_count` for string data?

Reply:
Changed to:
```c
    for i = 1, row_count do
        if field_type == 'unsigned' then
            for j = 1, column_count do
                if j % 2 == 1 then
                    tuple[j] = i
                else
                    tuple[j] = row_count - i + 1
                end
            end
        elseif field_type == 'string' then
            local char = string.char(97 + ((i - 1) % 26))
            local len = math.random(str_len_min, str_len_max)
            tuple = {i, string.rep(char, len)}
        else
            error("Unsupported field type")
        end
        s:insert(tuple)
```